live-examples/js-examples/map/map-prototype-entries.html

```diff
@@ -1,10 +1,11 @@
 <pre>
-<code id="static-js">var map1 = new Map();
+<code id="static-js">
```

Comment:
Adding the return after the opening `<code>` tag will add an extra blank line at the start of the example, which we don't want. I know this is quite non-obvious and when https://github.com/mdn/bob/pull/368 lands the problem will go away, but for now we should avoid it.

Reply:
I was afraid of that. :)  

Thanks.  Fixed.